source/_integrations/ecovacs.markdown

```diff
@@ -55,16 +55,16 @@ Additionally, **depending on your model**, the integration provides the followin
   - `Water amount`: Specify the water amount used during cleaning with the mop.
   - `Work mode`: Specify the mode, how the bot should clean.
 - **Sensor**:
-  - `Error`: The error code and a description of the error. `0` means no error. Disabled by default
-  - `Lifespan`: For each supported component an entity with the remaining lifespan will be created
-  - `Network`: The following network related entities will be created. All disabled by default
+  - `Error`: The error code and a description of the error. `0` means no error. Disabled by default.
+  - `Lifespan`: For each supported component, an entity with the remaining lifespan will be created.
+  - `Network`: The following network related entities will be created. All disabled by default.
     - `Ip address`
     - `Wi-Fi RSSI`
     - `Wi-Fi SSID`
   - `Cleaning cycle`:
     - `Area`: The cleaned area
     - `Time`: The cleaned time
-  -  `Total statistics`: Updated after each cleaning cycle:
```

Comment:
This one in particular fixes an indentation issue caused by the rogue space immediately after the list hyphen. Currently, those Area/Cleanings/Time metrics don't end up nested visually on the docs page like they appear here.